documentation/docs/25-build-and-deploy/60-adapter-cloudflare.md

```diff
@@ -30,6 +30,9 @@ export default {
 			routes: {
 				include: ['/*'],
 				exclude: ['<all>']
+			},
+			esbuild: {
+				external: [ 'fs' ]
 			}
```

Comment:
we've generally considered it an implementation detail that `esbuild` is used (e.g. in `adapter-node` we've switched between esbuild and rollup), so I don't think we'd want an option named `esbuild`
```suggestion
			external: [ 'fs' ]
```